src/cmd/cli/command/compose.go

```diff
@@ -45,6 +64,11 @@ func makeComposeUpCmd() *cobra.Command {
 
 			printPlaygroundPortalServiceURLs(deploy.Services)
 
+			var managedResources = GetUnreferencedManagedResources(deploy.Services)
+			if len(managedResources) > 0 {
+				term.Warnf("Defang cannot monitor status of the following managed service(s): %v.\n   Check if the managed service is up by checking your dependant service's status.", managedResources)
```

Comment:
I wonder if "your dependant service" is not as clear as it could be. Is this better?
```suggestion
				term.Warnf("Defang cannot monitor status of the following managed service(s): %v.\n   To check if the managed service is up, check the status of the service which depends on it.", managedResources)
```